docs/home/background.md

```diff
@@ -0,0 +1,71 @@
+# Background
+
+## Neutral Atom Qubits
+
+### Ground-Rydberg Coupling
```

Comment:
if we are only talking about ground rydberg, then just use a lv 2 title?

Reply:
Good catch, this was left over from when I originally thought it was worth mentioning we had hyperfine capabilities 👍